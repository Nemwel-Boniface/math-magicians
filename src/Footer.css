.footer {
  position: absolute;
  bottom: 0;
  width: 100%;
  padding: 18px;
  text-align: center;
}
.footer p {
  font-size: 20px;
}