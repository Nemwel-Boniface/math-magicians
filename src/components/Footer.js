const Footer = () => (
  <div className="footer">
    <p>
      &copy; Created by
      <a href="https://www.linkedin.com/in/nemwel-nyandoro-aa1b2620b/">Nemwel</a>
      {' '}
      under MIT licence.
    </p>
  </div>
);

export default Footer;
