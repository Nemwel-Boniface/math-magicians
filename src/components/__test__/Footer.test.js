import React from "react";
import { render, cleanup } from "@testing-library/react";
import Footer from '../Footer';

afterEach(cleanup)
it('Expect to create a Footer snapshot', () => {
  const { asFragment } = render(<Footer />)
  expect(asFragment(<Footer />)).toMatchSnapshot()
})