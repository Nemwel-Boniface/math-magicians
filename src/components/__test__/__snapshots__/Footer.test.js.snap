// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to create a Footer snapshot 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <p>
      Â© Created by
      <a
        href="https://www.linkedin.com/in/nemwel-nyandoro-aa1b2620b/"
      >
        Nemwel
      </a>
       under MIT licence.
    </p>
  </div>
</DocumentFragment>
`;
